scripts/prepdocs.py

```diff
@@ -29,7 +29,14 @@
 )
 from azure.storage.blob import BlobServiceClient
 from pypdf import PdfReader, PdfWriter
-from tenacity import retry, stop_after_attempt, wait_random_exponential
+from tenacity import (
+    retry,
+    retry_if_exception_type,
+    stop_after_attempt,
+    wait_random_exponential,
+)
+
+args = argparse.Namespace(verbose=False)
```

Comment:
I would prefer to not have args be a global at all, but at least this makes it possible to test.